packages/fastboot/test/fixtures/html-entrypoint/package.json

```diff
@@ -4,6 +4,30 @@
   "fastboot": {
     "schemaVersion": 5,
     "moduleWhitelist": [],
-    "htmlEntrypoint": "index.html"
+    "htmlEntrypoint": "index.html",
+    "config": {
```

Comment:
@ef4 @rwjblue should we put config in a `<meta name="<my-app>/config/fastboot-environment">` instead? Then we can merge with browser env and pass it to sandbox

Reply:
I haven't thought about this in a while but that seems fine to me.

Reply:
Done in latest commit. v5 schema after the fix is backwards compatible to embroider build